uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/shared/modules/constants.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-env amd */
+
+define([
+  'common/angular'
+], function (angular) {
+  'use strict';
+
+  angular.module('leave-absences.constants', [])
+    .constant('absence-type-colours', [
```

Comment:
why name this constant `absence-type-colours` and not `absenceTypeColours`?

Reply:
Good call. Should we uppercase them instead? https://github.com/johnpapa/angular-styleguide/issues/736

Reply:
Upper case is much better for constants. Please go ahead!